CMakeLists.txt

```diff
@@ -369,15 +389,19 @@ if (CLR_CMAKE_PLATFORM_UNIX)
 
   if(IS_64BIT_BUILD)
     add_definitions(-DBIT64=1)
-    add_definitions(-DFEATURE_PAL)
-  else (IS_64BIT_BUILD)
-    message(FATAL_ERROR "error: Detected non x86_64 target processor. Not supported!")
   endif(IS_64BIT_BUILD)
 
+  add_definitions(-DFEATURE_PAL)
```

Comment:
We are. Anything non-windows will use the PAL. So I think it is fine to define it across the board.